src/main/resources/changesets/changelog-consent-65.0.xml

```diff
@@ -0,0 +1,76 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="65.0" author="raejohanek">    
+     
+      <addColumn tableName="dacuser">
+        <column name="institution_id" type="bigint"/>
+      </addColumn>
+
+
+      <!--standardize institution names in user_property table so that all institution names exist in the institution table
+       and thus all dacusers can have institution ids assigned to them-->
+      <sql>
+      update user_property AS p SET propertyValue = 'Boehringer Ingelheim' WHERE p.propertyValue = 'Boehringer Ingelheim Pharma GmbH & Co KG';
```

Comment:
even though & is highlighted red it is not causing issues as a SQL query

Reply:
It is causing issues in the tests though. I think we need to escape it, will investigate.

Reply:
I see, maybe &amp; instead

Reply:
woah I did not type & it just turned it into that lol but thank you for bringing this to my attention I wasn't sure why the tests were failing

Reply:
The cleanest (I think) way to make this work is to replace the `&` with its `chr` value and concatenation, i.e., 

```
... WHERE p.propertyValue = 'Boehringer Ingelheim Pharma GmbH ' || chr(38) || ' Co KG';
```